cli/command/utils.go

```diff
@@ -2,6 +2,7 @@ package command
 
 import (
 	"bufio"
+	"context"
```

Comment:
Depending on if we support go 1.6 for `docker/cli` or not, it's no big deal in the *usage* of it (as the interface is the same)